zombie-monitor/src/main/scala/com/broadinstitute/dsp/zombieMonitor/DbReader.scala

```diff
@@ -128,8 +127,8 @@ object DbReader {
     override def markK8sClusterDeleted(id: Long): F[Unit] =
       markK8sClusterDeletedQuery(id.toInt).run.transact(xa).void
 
-    override def getk8sNodepoolsToDeleteCandidate: Stream[F, NodepoolToScan] =
-      activeNodepoolsQuery.stream.unNone.transact(xa)
```

Comment:
Why remove this?

Reply:
oh the `unNone`? 

I think the query to list `NodepoolToScan` records returned an option because it explicitly didn't handle Azure. So the stream here removed all the Nones. Now that we're using `Nodepool` we handle both clouds. 😀